code/modules/surgery/ear_surgery.dm

```diff
@@ -0,0 +1,77 @@
+//Head surgery to fix the ears organ
+/datum/surgery/ear_surgery
+	name = "Ear surgery"
+	requires_bodypart_type = NONE
+	organ_to_manipulate = ORGAN_SLOT_EARS
+	possible_locs = list(BODY_ZONE_HEAD)
+	steps = list(
+		/datum/surgery_step/incise,
+		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/saw,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/fix_ears,
+		/datum/surgery_step/close,
+	)
+
+//fix ears
+/datum/surgery_step/fix_ears
+	name = "fix ears (hemostat)"
+	implements = list(
+		TOOL_HEMOSTAT = 100,
+		TOOL_SCREWDRIVER = 45,
+		/obj/item/pen = 25)
+	time = 64
+
+/datum/surgery/ear_surgery/can_start(mob/user, mob/living/carbon/target)
+	var/obj/item/organ/internal/ears/target_ears = target.getorganslot(ORGAN_SLOT_EARS)
+	if(!target_ears)
+		to_chat(user, span_warning("It's hard to do surgery on someone's ears when [target.p_they()] [target.p_do()]n't have any."))
```

Comment:
Is this how other surgeries do it?? They should not be, this is going to spam the surgery tgui

Reply:
can_start should not have any side effects